tests/helpers/sharingHelper.js

```diff
@@ -147,9 +136,31 @@ const createFileInLastPublicShare = function (token, fileName) {
   })
 }
 
+/**
+ * convert JS object to form url encoded data.
+ *
+ * @param {object} obj
+ * @returns {string}
+ */
+const toFormUrlEncoded = function (obj) {
+  let toUpdate
```

Comment:
that variable name somehow does not have anything to do with the function name. maybe call it something like `formData`